### ISSUE TITLE
C# CC0013 CC0014 ternary operator fix removes comments

### ISSUE THREAD

**Issue by KristianWedberg** [code-cracker/code-cracker#921]
#Bug

C# CC0013 and CC0014 ternary operator fixes removes comments in code-cracker v1.0.3. 

This CC0014 code:

````csharp
int a;
if (Environment.MachineName == "Bilbo")
{
    a = 0; // Success
}
else
{
    a = 1; // Failure
}
````

Currently becomes this after fix applied:

````csharp
a = Environment.MachineName == "Bilbo" ? 0 : 1;
````

And this code:

````csharp
int a;
if (Environment.MachineName == "Bilbo")
    a = 0; // Success
else
    a = 1; // Failure
````

Currently becomes this after fix applied:

````csharp
int a;
a = Environment.MachineName == "Bilbo" ? 0 : 1; // Failure
````

Expected output in both above cases after fix applied:

````csharp
int a;
a = Environment.MachineName == "Bilbo" 
    ? 0 // Success
    : 1; // Failure
````

CC0013 behaves in the exact same (incorrect) way as CC0014.

As noted on other VB ternary operator fix issues, true and false comments should _not_ be combined, since each comment almost always only apply to one of the branches.

If 'fixing the fix' properly is non-trivial, I would suggest not creating any fix suggestion at all if there are comments, rather than silently removing them.

**Comment by carloscds** [comment]
@KristianWedberg Really is non-trivial fix. My suggestion is disable this fix if you use this kind of comment.

**Comment by KristianWedberg** [comment]
This issue _will_ certainly end up eating users comments, since few will know to look out for it. Doing this silently is... unfortunate. I would still recommend making CC0014 hidden until this issue is addressed.

**Comment by giggio** [comment]
I agree with you both, if there comments, we do not offer a diagnostic, for both CC0013 and CC0014.
